pkg/machines/components/vcpuModal.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import cockpit from 'cockpit';
+
+import { show_modal_dialog } from 'cockpit-components-dialog.jsx';
+import InfoRecord from './infoRecord.jsx';
+
+React;
+
+const _ = cockpit.gettext;
+
+const VCPUModalBody = ({ vcpus, cpu }) => (
+    <div className="modal-body">
+        <table className="vcpu-detail-modal-table">
+            <tr>
+                <td>
+                    <table className='form-table-ct'>
+                        <InfoRecord descr={_("Count")} value={vcpus.count}/>
+                        <InfoRecord descr={_("Placement")} value={vcpus.placement}/>
+                        <InfoRecord descr={_("Max")} value={vcpus.max}/>
```

Comment:
"Max" is shortform. I would suggest the full word, "Maximum" here, like in virt-manager.